Ds3/Runtime/HttpHelper.cs

```diff
@@ -25,6 +23,16 @@ internal static class HttpHelper
         };
 
         /// <summary>
+        /// Specified as "Unreserved" by the RFC
+        /// </summary>
+        private static readonly char[] _unreservedCharsParam = { '-', '.', '_', '~', '(', ')' };
```

Comment:
_hexChars is not used here anymore => can be removed
and static var should start with Capital